tests/epyccel/test_epyccel_sets.py

```diff
@@ -0,0 +1,48 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+import pytest
+from  pyccel.epyccel import epyccel
+
+@pytest.fixture( params=[
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="set methods not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="set methods not implemented in c"),
+            pytest.mark.c]),
+        pytest.param("python", marks = pytest.mark.python)
+    ],
+    scope = "module"
+)
+def language(request):
+    return request.param
+
+def test_add_1(language) :
+    def add_int():
+        a = {1,3,45}
+        a.add(4)
+        return a
+    epyc_add_element = epyccel(add_int, language = language)
+    pyccel_result = epyc_add_element()
+    python_result = add_int()
+    assert python_result == pyccel_result
+
+def test_add_element2(language) :
+    def add_complex():
```

Comment:
This I would call `test_add_literal_complex`.

Reply:
fixed : 0ff444f